backend/modules/auth/router.py

```
@@ -203,31 +205,57 @@ async def logout_user(response: Response):
+    # Email is sent via create_task so both branches (user found / not found)
+    # return immediately with the same latency profile.
+    if user and (not user.provider or user.provider == "password"):
+        token = auth_service.generate_password_reset_token(user.id)
+        reset_url = f"{settings.frontend_url}/reset-password?token={token}"
+        asyncio.create_task(
+            send_password_reset_email(
+                recipient_email=user.email,
+                user_name=user.full_name or user.username,
+                reset_url=reset_url,
+            )
+        )
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)